pkg/metrics/telemetry.go

```diff
@@ -475,7 +475,7 @@ func GetNonTransactionalStmtCounter() NonTransactionalStmtCounter {
 
 // GetSavepointStmtCounter gets the savepoint statement executed counter.
 func GetSavepointStmtCounter() int64 {
-	return readCounter(StmtNodeCounter.With(prometheus.Labels{LblType: "Savepoint", LblDb: ""}))
+	return readCounter(StmtNodeCounter.WithLabelValues("Savepoint", "", ""))
```

Comment:
If I understand correctly, `Savepoint` can be executed in any session, thus, pass resource-group label "" can result in incorrect result.  But since prometheus metrics does not provide a iter api, seems there is no good way to handle this

Reply:
yes